source/extensions/transport_sockets/s2a/tsi_frame_protector.cc

```diff
@@ -0,0 +1,116 @@
+#include "source/extensions/transport_sockets/s2a/tsi_frame_protector.h"
+
+#include "source/common/buffer/buffer_impl.h"
+#include "source/common/common/assert.h"
+
+#include "src/core/tsi/transport_security_grpc.h"
+#include "src/core/tsi/transport_security_interface.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace S2A {
```

Comment:
@asraa Are there any Envoy guidelines around this being "S2A" or "S2a"? "S2A" is definitely preferable.